.eslintrc

```diff
@@ -25,6 +25,7 @@
         'one-var':               ['error', { initialized: 'never', uninitialized: 'always' }],
         'semi':                  ['error', 'always'],
         'camelcase':             0,
+        "no-bitwise":            1,
```

Comment:
@binary-hamedanchi @negar-binary I hope this is not a problem?

Reply:
@4p00rv Since bitwise operators are rarely used, could you use single line disable like `// eslint-disable-line no-bitwise` instead, as this way, I'm concerned about mistyped `&&` or `||` to `&` or `|` in the future.
Thanks.